ci-operator/config/operator-framework/operator-marketplace/operator-framework-operator-marketplace-master.yaml

```diff
@@ -40,11 +40,11 @@ tests:
   commands: make unit
   container:
     from: src
-- as: e2e-gcp
+- as: e2e-gcp-ovn
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp
-- as: e2e-gcp-serial
+- as: e2e-gcp-ovn-serial
```

Comment:
We [recently](https://github.com/openshift/release/pull/37945) moved this to from gcp to aws due to some issues on that cloud provider.

Reply:
We only did this on master, but we should probably do this in all places.

Reply:
sure